.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Deploy to Netlify
+# This workflow runs periodically builds (daily) to deploy the
+# CometBFT documentation website to Netlify
+on:
+  pull_request:
```

Comment:
```suggestion
  push:
    branches:
      - main
```

Reply:
If you run this on `pull_request`, then it's going to deploy on every PR, which isn't what we want.

Reply:
lgtm, thanks.